apps/auth/Dockerfile

```
@@ -1,20 +1,35 @@
-FROM oven/bun:latest AS base
-RUN apt-get update -y && apt-get install -y openssl
+# syntax=docker/dockerfile:1
+
+FROM oven/bun:slim AS builder
+LABEL org.opencontainers.image.title="Zephyr Auth Service" \
+      org.opencontainers.image.description="Authentication service for Zephyr" \
+      org.opencontainers.image.vendor="Zephyr"
+RUN apt-get update && apt-get install -y openssl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)